### DISCUSSION TITLE
For public repositories, is there a monthly limit on minutes?

### DISCUSSION THREAD

**Discussion by mcarans** [community/community#26054] [Actions]
<p>My understanding of GitHub’s documentation is that while there are limits on number of GitHub Actions minutes per month for private repos (eg. 2000 for free plan), there are no limits on number of minutes per month for public repos regardless of plan ie. on the free plan, you get unlimited monthly GitHub Actions minutes for public repos. Is that correct?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/mcarans">@mcarans</a>,</p>
<p>Yes, your understanding is correct.<br>
GitHub Actions usage of minutes and storage is free for public repositories.</p>

**Reply by pavinduLakshan** [reply]
But still there can be rate limiting stuff, isn't it?

**Reply by madhuryadutta** [reply]
Still there is  a **execution limit** per **Job Run** .
I got the following **error** after running **github hosted runner for 6hour (360 minutes)** .
![Screenshot from 2024-01-08 00-20-24](https://github.com/community/community/assets/97863234/de42c738-48bf-4336-8c94-893e2afff791)


After searching for the reason I find the following Information
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes
![Screenshot from 2024-01-08 00-19-37](https://github.com/community/community/assets/97863234/80acda84-6fb9-4d40-8b70-6e115d195026)


 The following answer says that **self hosted runner** can be run upto **72 hour** and **github hosted runner** for **6 hour**.
https://github.com/orgs/community/discussions/27177
